interface/patient_file/summary/demographics.php

```diff
@@ -191,1287 +192,1009 @@ function image_widget($doc_id, $doc_catg)
-    }
+        /* Override font sizes in the theme. */
+        #DEM .groupname {
+            font-size: <?php echo attr($FONTSIZE); ?>rem;
+        }
 
-    #DEM .data td {
-        font-size: <?php echo attr($FONTSIZE); ?>rem;
-    }
+        #DEM .label {
+            font-size: <?php echo attr($FONTSIZE); ?>rem;
+        }
 
-    <?php } ?>
+        #DEM .data {
+            font-size: <?php echo attr($FONTSIZE); ?>rem;
+        }
 
-</style>
-<title><?php echo xlt("Dashboard{{patient file}}"); ?></title>
+        #DEM .data td {
+            font-size: <?php echo attr($FONTSIZE); ?>rem;
+        }
 
-<?php
-$arrOeUiSettings = array(
-    'heading_title' => xl('Medical Record Dashboard'),
-    'include_patient_name' => true,
-    'expandable' => false,
-    'expandable_files' => array(),//all file names need suffix _xpd
-    'action' => "",//conceal, reveal, search, reset, link or back
-    'action_title' => "",
-    'action_href' => "",//only for actions - reset, link or back
-    'show_help_icon' => true,
-    'help_file_name' => "medical_dashboard_help.php"
-);
-$oemr_ui = new OemrUI($arrOeUiSettings);
-?>
-</head>
+        <?php } ?> :root {
+            --white: #fff;
+            --bg: hsl(0 0% 90%);
+        }
 
-<body class="mt-3 patient-demographics">
-<div id="container_div" class="<?php echo $oemr_ui->oeContainer(); ?>">
-    <a href='../reminder/active_reminder_popup.php' id='reminder_popup_link' style='display: none;' onclick='top.restoreSession()'></a>
+        body {
+            background: var(--bg) !important;
+        }
 
-    <a href='../birthday_alert/birthday_pop.php?pid=<?php echo attr_url($pid); ?>&user_id=<?php echo attr_url($_SESSION['authUserID']); ?>' id='birthday_popup' style='display: none;' onclick='top.restoreSession()'></a>
-    <?php
+        section {
+            background: var(--white);
+            margin-top: .25em;
+            padding: .25em;
+        }
 
-    $thisauth = AclMain::aclCheckCore('patients', 'demo');
-    if ($thisauth) {
-        if ($result['squad'] && !AclMain::aclCheckCore('squads', $result['squad'])) {
-            $thisauth = 0;
+        .section-header-dynamic {
+            border-bottom: none;
         }
-    }
+    </style>
+    <title><?php echo xlt("Dashboard{{patient file}}"); ?></title>
+</head>
 
+<body class="mt-3 patient-demographic bg-light">
+
+    <?php
     // Create and fire the patient demographics view event
     $viewEvent = new ViewEvent($pid);
     $viewEvent = $GLOBALS["kernel"]->getEventDispatcher()->dispatch(ViewEvent::EVENT_HANDLE, $viewEvent, 10);
+    $thisauth = AclMain::aclCheckCore('patients', 'demo');
 
-    if (
-        !$thisauth ||
-        !$viewEvent->authorized()
-    ) {
-        echo "<p>(" . xlt('Demographics not authorized') . ")</p>\n";
-        echo "</body>\n</html>\n";
+    if (!$thisauth || !$viewEvent->authorized()) {
+        echo $twig->getTwig()->render('core/unauthorized.html.twig', ['pageTitle' => "Medical Dashboard"]);
```

Comment:
better to translate the pageTitle here via xl and then just html escape it in the twig
(always better to translate a constant rather than a variable, if possible, so then it will get picked up in the translation project pipeline)